api/apps/api/test/legacy-project-import/legacy-project-import.e2e-spec.ts

```diff
@@ -160,12 +196,27 @@ const getFixtures = async () => {
         }),
       );
     },
+    GivenSomePieceFailed: async (errors: string[] = []) => {
```

Comment:
Should we rename this fixture to `GivenFirstPieceFailed` or even `GivenFirstBatchFailed`